rustyline-derive/src/lib.rs

```diff
@@ -102,32 +102,42 @@ pub fn highlighter_macro_derive(input: TokenStream) -> TokenStream {
         quote! {
             #[automatically_derived]
             impl #impl_generics ::rustyline::highlight::Highlighter for #name #ty_generics #where_clause {
-                fn highlight<'l>(&self, line: &'l str, pos: usize) -> ::std::borrow::Cow<'l, str> {
-                    ::rustyline::highlight::Highlighter::highlight(&self.#field_name_or_index, line, pos)
+                #[cfg(any(not(feature = "split-highlight"), feature = "ansi-str"))]
```

Comment:
No `split-highlight` in this PR please

Reply:
I am sorry for that, I directly copy `rustyline-derive/src/lib.rs` for the old version. I only copy this single file and I will remove those :)

Reply:
No worry !